docs/user/dql/functions.rst

```diff
@@ -424,10 +478,37 @@ FLOOR
 Description
 >>>>>>>>>>>
 
-Specifications:
+Usage: FLOOR(a) takes the floor of value a.
 
-1. FLOOR(NUMBER T) -> T
+Argument type: a: double
```

Comment:
Same as above.
Could you add limitation too?

Reply:
Addressed in b5fd1208eb4d4012ce8a7fa16b93a2b50ac4eaf1